src/app/pages/plugins/plugin/plugin.component.html

```diff
@@ -2,6 +2,10 @@
     <section class="info">
         <div fxLayout="row">
             <div>
+                <section>PLUGIN</section>
```

Comment:
i've seen the `translate` pipe used in the templates a lot. maybe it's needed here?